### PR TITLE
Zenodo bip backlink

### LINKED ISSUE #633: Try to submit BIP Trial/Population back-link along with Zenodo metadata
The idea is to have a URL that leads back from Zenodo deposition page directly to BIP. Even better - to the exact submitted object (either a PlantTrial or a PlantPopulation).

Idea sent by Thomas Alcock and @teatree1212.

### PR DESCRIPTION
**Pull request by Nuanda**
Fixes #633

### CODE REVIEW

**Comment by Nuanda** [comment]
@kammerer To my surprise, something like that (see travis.yml) seems to work. I have repeated the build three times and it didn't fail :).

I will test again without the sleep.

File: app/decorators/submission_decorator.rb [hunk 1]
```diff
@@ -33,4 +28,22 @@ def doi
...
+  def bip_link
+    if object.submitted_object
+      h.data_tables_url(self_url_params)
+    end
+  end
+
+  private
+
+  def self_url_params
+    nil unless object.submitted_object
```
Comment (kammerer) [inline app/decorators/submission_decorator.rb]: Missing `return`?

File: app/decorators/submission_decorator.rb [hunk 2]
```diff
@@ -33,4 +28,22 @@ def doi
   def label
     raise Exception.new('Should be extended by subclasses')
   end
+
+  def bip_link
```
Comment (kammerer) [inline app/decorators/submission_decorator.rb]: It returns url, not link (like e.g. `link_to`), so `bip_url` (or `data_tables_url`, IMO better).

**Comment by kammerer** [comment]
I added missing `return` and removed some repeated code.
You may consider my suggestion about method name. Besides that, it's OK :+1:

**Pull request merged**